anndata/_core/anndata_base.py

```diff
@@ -0,0 +1,207 @@
+from abc import abstractmethod
+from typing import Tuple
+import pandas as pd
+
+from .file_backing import AnnDataFileManager
+from ..utils import DeprecationMixinMeta
+
+
+class AnnDataBase(metaclass=DeprecationMixinMeta):
+    def _init_as_actual(
```

Comment:
Ok, so perhaps then the way to go is to enforce `setters` as in https://github.com/scverse/anndata/pull/949#discussion_r1132591625 so that people have to do this?  I'm not sure why I didn't go with `setter`

Reply:
This way I don't have to enforce an "order" to how things are done.  I can throw an error if you don't implement an `obs` setter or the like.